src/ServiceControl.Contracts/FailedMessageArchived.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ServiceControl.Contracts
+{
+    /// <summary>
+    /// Event emitted for each failed message that was archived
+    /// </summary>
+    public class FailedMessageArchived
```

Comment:
We usually do bulk archiving of groups. Would it make more sense to emit a single event with a list of failed message ids?

Reply:
Would this be emitted with the group archived event?